DataFormats/Detectors/GlobalTracking/include/DataFormatsGlobalTracking/RecoContainer.h

```diff
@@ -199,6 +202,29 @@ struct RecoContainer {
   auto getITSTracksMCLabels() const { return tracksMCPool.getSpan<o2::MCCompLabel>(GTrackID::ITS); }
   auto getITSTrackMCLabel(GTrackID id) const { return tracksMCPool.get_as<o2::MCCompLabel>(id); }
 
+  // MFT
+  template <typename U> // o2::mft::TrackMFT
+  auto getMFTTracks() const
+  {
+    return fwdTracksPool.getSpan<U>(GTrackID::MFT);
+  }
+
+  template <typename U> // o2::mft::TrackMFT or TrackParCovFwd
+  auto getMFTTrack(GTrackID id) const
+  {
+    return fwdTracksPool.get_as<U>(id);
+  }
+
+  template <typename U> // o2::itsmft::ROFRecord
+  auto getMFTTracksROFRecords() const
+  {
+    return tracksROFsPool.getSpan<U>(GTrackID::MFT);
+  }
+
+  auto getMFTTracksClusterRefs() const { return clusRefPool.getSpan<int>(GTrackID::MFT); }
+  auto getMFTTracksMCLabels() const { return tracksMCPool.getSpan<o2::MCCompLabel>(GTrackID::MFT); }
+  auto getMFTTrackMCLabel(GTrackID id) const { return tracksMCPool.get_as<o2::MCCompLabel>(id); }
+
```

Comment:
And you can simply copy-paste MFT part from here, just forward-declare your types: 
https://github.com/AliceO2Group/AliceO2/blob/e79e69c0ed6bc54834217eea71e2f2721e7fa662/DataFormats/Detectors/GlobalTracking/include/DataFormatsGlobalTracking/RecoContainer.h#L287-L296